flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/typeutils/AvroSerializerSnapshotTest.java

```diff
@@ -44,11 +44,11 @@
 import static org.apache.flink.api.common.typeutils.TypeSerializerMatchers.isCompatibleAfterMigration;
 import static org.apache.flink.api.common.typeutils.TypeSerializerMatchers.isCompatibleAsIs;
 import static org.apache.flink.api.common.typeutils.TypeSerializerMatchers.isIncompatible;
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.HamcrestCondition.matching;
```

Comment:
I'm not sure whether this is the right change at this time, while the `org.apache.flink.api.common.typeutils.TypeSerializerMatchers` are still being used.  This permits AssertJ to use the old style matchers for now, but the class will need to be refactored if they are turned into AssertJ `Conditions`.

Reply:
Looks fine to me.